enterprise/frontend/src/embedding-sdk/bin/generate-nextjs-compat.ts

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable no-console */
+import fs from "fs";
+
+import path from "path";
+import prettier from "prettier";
+import { match } from "ts-pattern";
+
+type ComponentDefinition = {
+  mainComponent: string;
+  subComponents: string[];
+};
+
+// START OF CONFIGURATION
+// Note: this list needs to be updated when new components are added
+
+const COMPONENTS_TO_EXPORT: ComponentDefinition[] = [
+  // eslint-disable-next-line no-literal-metabase-strings -- cli tool
```

Comment:
Oh, I thought we had a warning about unused eslint-disable, I had to put it because it was complaining about `Metabase`Provider, but that line is now commented out